### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by locyi** [termux/termux-app#3475] [Q&A] [answered]
When I open the software, I see 'exec ("/data/data/com.termux/files/usr/bin/login"): Permission denied).
[Process completed (code 1)-press Enter] '

**Comment by sylirre** [comment, marked as answer]
Dif you install the Termux on SD card? Seems like the case. Please check this and if so, move the Termux app to internal storage and clear data.

Adoptable storage also counts as "installation on sd card".

**Comment by locyi** [comment]
I didn't install it in SD card.Thanks😄

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
